### ISSUE TITLE
a11y in the one-off SaaS buttons in Status Page

### ISSUE THREAD

**Issue by eckdanny-osi** [ca-cwds/design-system#116]
The github and slack buttons dont have an `alt` prop.

**Comment by eckdanny-osi** [comment]
closed in #196
